### PR TITLE
Extensions - Remove Details from Documentation

### PR DESCRIPTION
**Pull request by sunilarjun**
The primary goal of this PR is to remove the details page mentions from the extensions UI page and the how-to guide for installing/uninstalling extensions. Made some minor language changes as well due to the update, and will be updating the screenshots with a subsequent PR for the respective operating systems. This work is tied to [issue 4575](https://github.com/rancher-sandbox/rancher-desktop/issues/4575) and [issue 4741](https://github.com/rancher-sandbox/rancher-desktop/issues/4741).

### CODE REVIEW

File: docs/how-to-guides/installing-uninstalling-extensions.md [hunk 1]
```diff
@@ -21,13 +21,9 @@ There are two ways in which you can install extensions, a method using the UI an
 
 #### Using the UI
 
-1. Click **Extensions** from the main UI to navigate to the **Catalog** tab. Here you can search through extensions available for use.
+Click **Extensions** from the main UI to navigate to the **Catalog** tab. Here you can search through extensions available for use which can be downloaded using the **Install** button.
```
Comment (gunamata) [inline docs/how-to-guides/installing-uninstalling-extensions.md:24]:
```suggestion
Click **Extensions** from the main UI to navigate to the **Catalog** tab. Here you can search through extensions available for use which can be downloaded and installed using the **Install** button.
```

File: docs/how-to-guides/installing-uninstalling-extensions.md [hunk 2]
```diff
@@ -48,13 +44,9 @@ rdctl extension install <image-id>:<tag>
 
 #### Using the UI
 
-1. Click **Extensions** from the main UI to navigate to the **Catalog** tab. Here you can search through extensions available for use.
-
-![](https://suse-rancher-media.s3.us-east-1.amazonaws.com/desktop/1.9-tech-preview/how-to/macOS_ExtensionsCatalog.png)
+Click **Extensions** from the main UI to navigate to the **Catalog** tab. Here you can search through extensions available for use which can be downloaded using the **Install** button.
```
Comment (gunamata) [inline docs/how-to-guides/installing-uninstalling-extensions.md:47]:
```suggestion
Click **Extensions** from the main UI to navigate to the **Catalog** tab. Here you can search through extensions available for use which can be downloaded and installed using the **Install** button.
```

File: docs/how-to-guides/installing-uninstalling-extensions.md [hunk 3]
```diff
@@ -75,13 +67,9 @@ rdctl extension install <image-id>:<tag>
 
 #### Using the UI
 
-1. Click **Extensions** from the main UI to navigate to the **Catalog** tab. Here you can search through extensions available for use.
-
-![](https://suse-rancher-media.s3.us-east-1.amazonaws.com/desktop/1.9-tech-preview/how-to/Linux_Extensions.png)
+Click **Extensions** from the main UI to navigate to the **Catalog** tab. Here you can search through extensions available for use which can be downloaded using the **Install** button.
```
Comment (gunamata) [inline docs/how-to-guides/installing-uninstalling-extensions.md:70]:
```suggestion
Click **Extensions** from the main UI to navigate to the **Catalog** tab. Here you can search through extensions available for use which can be downloaded and installed using the **Install** button.
```

File: docs/how-to-guides/installing-uninstalling-extensions.md [hunk 4]
```diff
@@ -109,11 +97,9 @@ Additionally, there are two ways in which you can uninstall extensions, a method
 
 #### Using the UI
 
-1. Click **Extensions** from the main UI to navigate to the **Catalog** tab, then click on the **Installed** tab.
-
-2. Once under the **Installed** tab, you can uninstall extensions by clicking on the **Uninstall** button on the right hand side as pictured below.
+Click **Extensions** from the main UI to navigate to the **Catalog** tab. Here you can search through extensions available for use, and extensions which have been downloaded can be uninstalled from this view by clicking the **Remove** button.
```
Comment (gunamata) [inline docs/how-to-guides/installing-uninstalling-extensions.md:100]:
```suggestion
Click **Extensions** from the main UI to navigate to the **Catalog** tab. On this view, you can search through the available extensions, and uninstall the already installed extensions.
```

File: docs/how-to-guides/installing-uninstalling-extensions.md [hunk 5]
```diff
@@ -134,11 +120,9 @@ rdctl extension uninstall <image-id>:<tag>
 
 #### Using the UI
 
-1. Click **Extensions** from the main UI to navigate to the **Catalog** tab, then click on the **Installed** tab.
+Click **Extensions** from the main UI to navigate to the **Catalog** tab. Here you can search through extensions available for use, and extensions which have been downloaded can be uninstalled from this view by clicking the **Remove** button.
```
Comment (gunamata) [inline docs/how-to-guides/installing-uninstalling-extensions.md:123]:
```suggestion
Click **Extensions** from the main UI to navigate to the **Catalog** tab. On this view, you can search through the available extensions, and uninstall the already installed extensions.
```

File: docs/how-to-guides/installing-uninstalling-extensions.md [hunk 6]
```diff
@@ -159,11 +143,9 @@ rdctl extension uninstall <image-id>:<tag>
 
 #### Using the UI
 
-1. Click **Extensions** from the main UI to navigate to the **Catalog** tab, then click on the **Installed** tab.
-
-2. Once under the **Installed** tab, you can uninstall extensions by clicking on the **Uninstall** button on the right hand side as pictured below.
+Click **Extensions** from the main UI to navigate to the **Catalog** tab. Here you can search through extensions available for use, and extensions which have been downloaded can be uninstalled from this view by clicking the **Remove** button.
```
Comment (gunamata) [inline docs/how-to-guides/installing-uninstalling-extensions.md:146]:
```suggestion
Click **Extensions** from the main UI to navigate to the **Catalog** tab. On this view, you can search through the available extensions, and uninstall the already installed extensions.
```

**Review by gunamata — CHANGES_REQUESTED** [review]
I made some minor proposals to improve readability, avoid ambiguity etc. I may not have modified in all the versions. If you agree with my proposals, I will let you apply the changes at all required places.

**Comment by sunilarjun** [comment]
Thanks for the review Guna! Updated the language with the proposed changes.

**Review by gunamata — APPROVED** [review]
LGTM, Thank you

**Pull request merged**